src/dashboard/src/components/ingest/views_as.py

```diff
@@ -16,9 +16,17 @@
 logger = logging.getLogger('archivematica.dashboard')
 
 
+class ArchivesSpaceConfigurationException(Exception):
```

Comment:
You could make it a subclass of `ArchivesSpaceError` or just raise `ArchivesSpaceError` considering that the outcome is the same regardless.

If you decide to stick with `ArchivesSpaceConfigurationException` I think that you should probably call it `ArchivesSpaceConfigurationError` instead, that seems to be the naming convention.

Reply:
I went for ArchivesSpaceError in the end! Good call.